src/tests/components/registerComponent.test.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { shallow } from "enzyme/build";
+import RegisterComponent from "../../components/registerComponent";
+
+describe("login component", () => {
+  let wrapper;
+  let props = {
+    username: "",
+    email: "",
+    password: "",
+    errors: {
+      username: "",
+      email: "",
+      password: ""
+    },
+    error_message: {
+      errors: {}
+    },
+    classes: null,
+    handleOnChange: jest.fn(),
+    handleOnSubmit: jest.fn()
+  };
+  beforeEach(() => {
+    wrapper = shallow(<RegisterComponent {...props} />);
+  });
+  it("should render with fail", () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+  it("should simulate  onchange", () => {
+    const onChange = wrapper.dive();
+    onChange.find("#username").simulate("change");
+    expect(props.handleOnChange).toHaveBeenCalled();
+  });
+  it("should simulate have been clicked", () => {
+    const onSubmit = wrapper.dive();
+    onSubmit.find(".button-success").simulate("click");
```

Comment:
Every test should have an expectation or assertion (this applies for every other test without an `except` statement